bin/zf-maintainer

```diff
@@ -2,7 +2,7 @@
 <?php
 /**
  * @see       https://github.com/zendframework/maintainers for the canonical source repository
- * @copyright Copyright (c) 2017-2018 Zend Technologies USA Inc. (http://www.zend.com)
+ * @copyright https://github.com/zendframework/maintainers/blob/master/COPYRIGHT.md Copyright
```

Comment:
Here and all others referencing the repo's copyright file: we do not need the word "Copyright" at the end; it's already in the annotation name.